spec/dealership_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rspec'
+require './lib/car'
+require './lib/dealership'
+
+RSpec.describe Dealership do
+  it 'IT2 p2' do
+    dealership = Dealership.new("Acme Auto", "123 Main Street")
+    expect(dealership.inventory).to eq([])
+    expect(dealership.inventory_count).to eq(0)
+  end
+
+  it 'IT2 p2' do
+    dealership = Dealership.new("Acme Auto", "123 Main Street")
+    car_1 = Car.new("Ford Mustang", 1500, 36)
+    car_2 = Car.new("Toyota Prius", 1000, 48)
+    dealership.add_car(car_1)
+    dealership.add_car(car_2)
+    expect(dealership.inventory).to eq([car_1, car_2])
+    expect(dealership.inventory_count).to eq(2)
+  end
+
+  it 'IT3 p1' do
+    dealership = Dealership.new("Acme Auto", "123 Main Street")
+    dealership.has_inventory?
+    expect(dealership.has_inventory?).to eq(false)
+  end
+
+  it 'IT3 p2' do
+    dealership = Dealership.new("Acme Auto", "123 Main Street")
+    car_1 = Car.new("Ford Mustang", 1500, 36)
+    car_2 = Car.new("Toyota Prius", 1000, 48)
+    car_3 = Car.new("Toyota Tercel", 500, 48)
+    car_4 = Car.new("Chevrolet Bronco", 1250, 24)
+    dealership.add_car(car_1)
+    dealership.add_car(car_2)
+    dealership.add_car(car_3)
+    dealership.add_car(car_4)
+    expect(dealership.cars_by_make("Toyota")).to eq([car_2, car_3])
+    expect(dealership.cars_by_make("Ford")).to eq([car_1])
+    expect(dealership.total_value).to eq(156000)
+    #expect(dealership.details).to eq({15600, "123 Main Street"})
+  end
+
+  it 'IT4 p1' do
+    dealership = Dealership.new("Acme Auto", "123 Main Street")
+    car_1 = Car.new("Ford Mustang", 1500, 36)
+    car_2 = Car.new("Toyota Prius", 1000, 48)
+    car_3 = Car.new("Toyota Tercel", 500, 48)
+    car_4 = Car.new("Chevrolet Bronco", 1250, 24)
+    dealership.add_car(car_1)
+    dealership.add_car(car_2)
+    dealership.add_car(car_3)
+    dealership.add_car(car_4)
+    dealership.total_value
+    expect(dealership.average_price_of_car).to eq("39000")
+    expect(dealership.cars_sorted_by_price).to eq([car_3, car_2, car_4, car_1])
```

Comment:
Unfortunately, this isn't the correct order as we want to be sorting on total cost so it should be [car_3, car_4, car_2, car_1]